config/settings.d/dns_virsh.yml.example

```diff
@@ -0,0 +1,4 @@
+---
+#
+# Configuration file for libvirtd-specific 'virsh' dns provider
+#
```

Comment:
Perhaps just note that the only config is currently in the main settings.yml (the virsh network name, IIRC)